src/formatter.js

```diff
@@ -216,9 +216,7 @@ export class Formatter {
     const beams = options.auto_beam ? Beam.applyAndGetBeams(voice) : [];
 
     // Instantiate a `Formatter` and format the notes.
-    new Formatter()
-      .joinVoices([voice])
-      .formatToStave([voice], stave, { align_rests: options.align_rests, stave });
+    new Formatter().joinVoices([voice]).formatToStave([voice], stave, { align_rests: options.align_rests, stave });
```

Comment:
This change is made by es-lint autofix. This is really a strange rule to have this kind of chain in one line. IMO unreadable.